client/post-editor/post-editor.jsx

```diff
@@ -97,9 +97,26 @@ const messages = {
 				comment: 'Editor: Message displayed when a post is published privately, with a link to the site it was published on.'
 			} );
 		},
+		scheduled: function() {
+			var site = this.props.sites.getSelectedSite();
+
+			if ( ! site ) {
+				return i18n.translate( 'Post scheduled!' );
+			}
+
+			return i18n.translate( 'Post scheduled on {{siteLink/}}!', {
+				components: {
+					siteLink: <a href={ site.URL } target="_blank">{ site.title }</a>
+				},
+				comment: 'Editor: Message displayed when a post is scheduled, with a link to the site it was published on.'
+			} );
+		},
 		view: function() {
 			return i18n.translate( 'View Post' );
 		},
+		viewPreview: function() {
+			return i18n.translate( 'Preview Post' );
```

Comment:
This has me thinking about upcoming custom post type support, as there's not a label to define [when registering a custom post type](https://codex.wordpress.org/Function_Reference/register_post_type#labels) for "Preview Post" ([example API response](https://public-api.wordpress.com/rest/v1.1/sites/example.wordpress.com/post-types)). So while we could hard-code these for known post types (posts, pages), we may need to fall back to the "View" label for custom post types, which makes me question whether we should distinguish these labels in the first place. Is "View Post" confusing in the context of a scheduled post?